src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -17,8 +18,22 @@
  */
 class HTMLEditorSanitiser
 {
+    use Configurable;
     use Injectable;
 
+    /**
+     * rel attribute to add to link elements which have a target attribute (usually "_blank")
+     * this is to done to prevent reverse tabnabbing - see https://www.owasp.org/index.php/Reverse_Tabnabbing
+     * noopener includes the behaviour we want, though some browsers don't yet support it and rely
+     * upon using noreferrer instead - see https://caniuse.com/rel-noopener for current browser compatibility
+     *
+     * @var string
+     */
+    private static $link_noopener_value = 'noopener, noreferrer';
```

Comment:
This config prop name could probably be `$link_rel_value` since you're defining the value of the `rel` attribute

Reply:
have changed